### PR TITLE
CLDR-14649 archiver for tr35 .md files

### PR DESCRIPTION
**Pull request by srl295**
- see tools/scripts/tr-archive/README.md for instructions

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14649
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/scripts/tr-archive/archive.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const fs = require('fs').promises;
+const marked = require("marked");
+const jsdom = require("jsdom");
```
Comment (styfle) [inline tools/scripts/tr-archive/archive.js:3]: Not a big deal but you have mixed double quotes and single quotes 🙂

Comment (srl295) [reply tools/scripts/tr-archive/archive.js:3]: @styfle thanks! I'll run the prettier on the file

File: tools/scripts/tr-archive/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Prerequisites
+
+Node.js (Tested with v12)
+
+## How to use
+
+```shell
+$ bash make-tr-archive.html
+```
```
Comment (btangmu) [inline tools/scripts/tr-archive/README.md:17]: Should ".html" here be ".sh"?

**Review by btangmu — APPROVED** [review]
possible typo in the readme, see comment; otherwise lgtm

File: tools/scripts/tr-archive/README.md (hunk 2 above)
Comment (srl295) [reply tools/scripts/tr-archive/README.md:17]: fixed

**Comment by srl295** [comment]
rebased on maint

**Review by btangmu — APPROVED** [review]

**Pull request merged**